src/Mvc/test/Microsoft.AspNetCore.Mvc.Core.Test/ApplicationParts/ApplicationAssembliesProviderTest.cs

```diff
@@ -74,34 +74,6 @@ public void ResolveAssemblies_ReturnsRelatedAssembliesOrderedByName()
             Assert.Equal(new[] { ThisAssembly, assembly2, assembly1, assembly3 }, result);
```

Comment:
Removed several tests from this class that were either 1. not valuable or 2. no longer valid.